tests/profiling/test_profiler.py

```diff
@@ -188,9 +188,9 @@ def test_tracer_url_https():
 
 def test_tracer_url_uds():
     t = ddtrace.Tracer()
-    t.configure(hostname="foobar", https=True, uds_path="/foobar")
+    t.configure(uds_path="/foobar")
```

Comment:
Can we add another test that includes the hostname previously used for the UDS as this PR also adds logic for reusing UDS connection info?

Reply:
Yep good idea.